`dpkg-buildpackage` fails when building kernel packages

When running `build-debian.sh`, I get this error at the end of kernel compilation:

```
make[4]: *** [Makefile:357: __build_one_by_one] Error 2
make[3]: *** [debian/rules:8: build-arch] Error 2
dpkg-buildpackage: error: debian/rules binary subprocess returned exit status 2
make[2]: *** [scripts/Makefile.package:114: bindeb-pkg] Error 2
make[1]: *** [Makefile:1656: bindeb-pkg] Error 2
make: *** [Makefile:357: __build_one_by_one] Error 2
```

How does one fix this issue?

--- a/build-debian.sh
+++ b/build-debian.sh
@@ -2,7 +2,7 @@
 ACSOPATCH_VER=6.3
 KERNEL_VER=6.3
 sudo apt install build-essential libncurses5-dev fakeroot xz-utils libelf-dev liblz4-tool \
-  unzip flex bison bc debhelper rsync libssl-dev:native
+  unzip flex bison bc debhelper rsync libssl-dev:native dwarves
 wget -N https://raw.githubusercontent.com/benbaker76/linux-acs-override/main/$ACSOPATCH_VER/acso.patch
 wget -N https://github.com/torvalds/linux/archive/refs/tags/v$KERNEL_VER.zip
 unzip -o v$KERNEL_VER.zip

